tests/unit/test_runner.py

```diff
@@ -249,6 +255,57 @@ def test_runner_ignores_request_if_jira_is_linked_but_without_whiteboard(
     assert str(exc_info.value) == "no bug whiteboard matching action tags: devtest"
 
 
+@pytest.mark.asyncio
+async def test_execute_or_queue_happy_path(
+    mock_queue, 
+    bugzilla_webhook_request,
+):
+    mock_queue.is_blocked.return_value = False
+    await execute_or_queue(
+        request=bugzilla_webhook_request,
+        queue=mock_queue,
+        actions=mock.MagicMock(spec=Actions)
```

Comment:
why not use the `actions` fixture?

Reply:
Looking at the other tests that use the `actions` fixture, that would require mocking deeper things (like get_bug and get_issue) right? Since we're just testing the `execute_or_queue` logic here this seemed like the path of least resistance.